docusaurus/docs/reactnative/guides/channel_background_customization.mdx

```diff
@@ -0,0 +1,318 @@
+
+```tsx
+import { Channel, MessageInput, MessageList, ThemeProvider } from 'stream-chat-react-native';
+import { ImageBackground } from 'react-native';
+
+const IMAGE_URI =
+  'https://images.unsplash.com/photo-1549125764-91425ca48850?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxleHBsb3JlLWZlZWR8NjF8fHxlbnwwfHx8fA%3D%3D&auto=format&fit=crop&w=800&q=60';
+
+const ChannelScreen = ({ channel }) => (
+  <ThemeProvider style={theme}>
+    <Channel channel={channel}>
+      <ImageBackground
+        style={{ flex: 1 }}
+        source={{
+          uri: IMAGE_URI,
+        }}
+      >
+        <MessageList />
+        <MessageInput />
+      </ImageBackground>
+    </Channel>
+  </ThemeProvider>
+);
+
+export const theme = {
+  messageList: {
+    container: {
+      backgroundColor: 'transparent',
+    },
+  },
+};
+```
+
+---
+
+## Custom Background With Selection Screen
+
+In this step, we will add a button that will navigate to a separate screen, where the user will be able to select his
+favorite background image for a specific channel.
+
+<table>
+  <tr>
+    <td align='center' width='50%'>
+      <img src='https://user-images.githubusercontent.com/25864161/168006715-acae4b85-00cb-4b45-a127-aa8f94c13895.png' />
+    </td>
+    <td align='center' width='50%'>
+      <img src='https://user-images.githubusercontent.com/25864161/168006193-8fd4ad85-7553-4956-a7c6-5d6979e15ee4.png' />
+    </td>
+  </tr>
+  <tr></tr>
+  <tr>
+    <td align='center'>
+      <strong>Chat screen with customize background button</strong>
+    </td>
+    <td align='center'>
+      <strong>Wallpaper overview screen with background image options</strong>
+    </td>
+  </tr>
+</table>
+
+### Store and manage channel preferences
+
+To persist the channel preferences (at the moment, only the URI of the selected background), we will need to store some data.
+We will do it by using `react-native-mmkv`, a key-value storage [framework](https://github.com/mrousavy/react-native-mmkv#mmkv).
+
+Follow the [installation steps](https://github.com/mrousavy/react-native-mmkv#installation), and let's get started, shall we?
+
+We will start by creating our `ChannelBackgroundView` component.
+This component will be in charge of rendering the custom background by retrieving it from our key-value store.
+We save an object to a key-value store to be scalable and future-proof.
+You'd might want to add other preferences later, such as dimming value, background color, etc.
+
+```tsx
+import type { ViewProps } from 'react-native';
+import { useMMKVObject } from 'react-native-mmkv';
+
+type ChannelPreferences = {
+  imageUri: string;
+};
+
+const DEFAULT_BACKGROUND_URI = 'https://i.redd.it/3jfjc53fsyb61.jpg';
+
+const ChannelBackgroundView = ({
+  channelId,
+  ...props
+}: {
+  channelId: string;
+} & ViewProps) => {
+  const [channelPreferences] = useMMKVObject<ChannelPreferences>(channelId);
+  let uri: string | undefined = channelPreferences?.imageUri || DEFAULT_BACKGROUND_URI;
```

Comment:
```suggestion
  const uri: string | undefined = channelPreferences?.imageUri || DEFAULT_BACKGROUND_URI;
```

Reply:
Isn't the `| undefined` unnecessary here? I'd also avoid explicitly typing things as `undefined` and rather use something more explicit like `null`